tests/codeception/acceptance/AuthenticatedPermissionsCest.php

```diff
@@ -0,0 +1,147 @@
+<?php
+
+/**
+ * Test the restrictions on authenticated users.
+ */
+class AuthenticatedPermissionsCest {
+
+  /**
+   * Set up a file to test PHP injection.
+   */
+  public function _before(AcceptanceTester $I) {
+    file_put_contents(codecept_data_dir() . '/injection.php', '<?php echo("injection test"); die(); ?>');
```

Comment:
OOOOOH... I like this.

I am going to steal this for another issue I have been having with the lockup logo. @pookmish

Reply:
See: https://github.com/SU-SWS/stanford_profile/pull/220/files#diff-57971af0c9829dc91faff7b65b36951cR300-R308

Reply:
huh that could be a nice way to get images for usage.